docs/src/details/integration/driver/display/nxp_elcdif.rst

```diff
@@ -0,0 +1,57 @@
+.. _nxp_elcdif:
+
+=============
+NXP eLCDIF
+=============
+
+Overview
+--------
+eLCDIF is a peripheral that is provided on some of the NXP devices capable to drive display panels through
+the RGB interface, it supports different color depths and, on MIPI-DSI capable devices, its output ca be
+directed to the MIPI display physical interface. The LVGL's NXP eLCDIF driver is responsible to bind the
+NXP MCUx SDK low-level driver to the LVGL display subsystem. 
+
+
+Prerequisites
+-------------
+
+- This driver relies on the presence of the MCUx SDK from NXP in the same project
+- Activate the diver by setting :c:macro:`LV_USE_NXP_ELCDIF` to ``1`` in your *"lv_conf.h"*.
+
+Usage
+-----
+
+The LVGL driver for eLCDIF assumes the platform already configured the display low-level driver, 
+set the pin-mux, clocks, etc. It also requires the base address of the peripheral and configuration
+structure already set.
+
+The following code demonstrates using the diver in :cpp:enumerator:`LV_DISPLAY_RENDER_MODE_DIRECT` mode.
+
+.. code-block:: c
+    
+    elcdif_rgb_mode_config_t config;
+    ELCDIF_RgbModeGetDefaultConfig(&config);
+
+    lv_display_t * g_disp = lv_nxp_display_elcdif_create_direct(LCDIF, config, buffer1, buffer2, buf_size);
```

Comment:
What is `buffer1` and `buffer2`; How to allocate them? What should be `buf_size`?

Please mention that they can be just memory address too in external RAM.